net/vpnbypass/files/vpnbypass.init

```diff
@@ -127,11 +132,15 @@ stop_service() {
 
 service_triggers_load_interface() { is_supported_interface "$1" && ifaces="${ifaces}${1} "; }
 service_triggers() {
-		local ifaces n
-		config_load network; config_foreach service_triggers_load_interface 'interface';
-		procd_add_reload_trigger 'firewall' 'openvpn' 'vpnbypass'
-		procd_open_trigger
-			for n in $ifaces; do procd_add_reload_interface_trigger "$n"; procd_add_interface_trigger "interface.*" "$n" /etc/init.d/vpnbypass reload; done;
-			output "$serviceName monitoring interfaces: $ifaces\\n"
-		procd_close_trigger
+	local ifaces n
+	config_load network; config_foreach service_triggers_load_interface 'interface';
+	procd_open_trigger
+		procd_add_reload_trigger 'openvpn'
+		if type procd_add_service_trigger 1>/dev/null 2>&1; then
+			procd_add_service_trigger "service.restart" "firewall" /etc/init.d/${packageName} reload
```

Comment:
As I know the `procd_add_service_trigger` is not merged in openwrt master!

Reply:
Not to say that this should be merged before the dependent PR is merged, but with the code above the `procd_add_service_trigger` will be used in the init script as soon as it's available on the system. :)